pages/configuration/web_interface.rst

```diff
@@ -134,22 +134,22 @@ NGINX
 
     server
     {
-      listen      80 default_server;
-      listen      [::]:80 default_server ipv6only=on;
-      server_name graylog.example.org;
-
-      location /
-        {
-            proxy_set_header    Host $http_host;
-            proxy_set_header    X-Forwarded-Host $host;
-            proxy_set_header    X-Forwarded-Server $host;
-            proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
-            proxy_set_header    X-Graylog-Server-URL http://graylog.example.org/api;
-            proxy_pass          http://127.0.0.1:9000;
+        listen 80 default_server;
+        listen [::]:80 default_server ipv6only=on;
+
+        location / {
+          proxy_set_header Host $http_host;
+          proxy_set_header X-Forwarded-Host $host;
+          proxy_set_header X-Forwarded-Server $host;
+          proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+          proxy_set_header X-Graylog-Server-URL http://$server_addr/api;
```

Comment:
Wouldn't `$server_name` make more sense here? At least it would be clearly defined, in contrast to `$server_addr` which is simply *one* address of the server but not necessarily the canonical one (or the one the user wants to use).

➡️  https://nginx.org/en/docs/http/ngx_http_core_module.html